react/features/base/conference/reducer.ts

```diff
@@ -596,10 +596,7 @@ function _setRoom(state: IConferenceState, action: AnyAction) {
      */
     return assign(state, {
         error: undefined,
-        localSubject: undefined,
-        pendingSubjectChange: undefined,
-        room,
-        subject: undefined
+        room
     });
```

Comment:
Why do you drop these? Isn't this rest of state when room changes in mobile? Especially the pendingSubjectChange.

Reply:
Same thing.

Reply:
What if there is no auth and a user joins a different meeting? That's the problem that code tries to solve.